clients/docs/csharp.rst

```diff
@@ -0,0 +1,132 @@
+.. _client-examples-csharp:
+
+.NET Example
+============
+
+Produce messages to and consume messages from a |ak| cluster using the .NET Producer and Consumer.
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  `.NET Core 2.1 <https://dotnet.microsoft.com/download>`__ or higher to run the demo application
+
+-  On Windows, default trusted root CA certificates - which are required for secure access to Confluent Cloud - are stored in the Windows Registry. The .NET library does not currently have the capability to access these certificates, so you will need to obtain them from somewhere else, for example use the cacert.pem file distributed with curl (`download cacert.pm <https://curl.haxx.se/ca/cacert.pem>`__).
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. Clone the `confluentinc/examples GitHub repository <https://github.com/confluentinc/examples>`__ and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for .NET.
+
+   .. sourcecode:: bash
+
+      cd clients/cloud/csharp/
+
+.. include:: includes/client-example-create-file.rst
```

Comment:
currently the numbering isn't rendering properly - i think this will fix. 
![image](https://user-images.githubusercontent.com/11722533/89078886-16110700-d33a-11ea-867e-16598215633f.png)


```suggestion
#. .. include:: includes/client-example-create-file.rst
```

Reply:
@joel-hamill I had noticed the odd numbering but didn't know how to fix.  Thanks for the suggestion.